polkit/Makefile

```diff
@@ -0,0 +1,30 @@
+sysconfdir      ?= /etc
```

Comment:
I think that packages should be installing rules in `/usr/share/polkit-1/rules.d` instead, as `/etc/polkit-1/rules.d` is supposed to be for user written rules?

Reply:
Yeah, that's probably a good call.